scripts/globals/avatars_favor.lua

```diff
@@ -0,0 +1,149 @@
+-----------------------------------
+-- Avatars Favor helper
+-----------------------------------
+require("scripts/globals/pets")
+require("scripts/globals/status")
+require("settings/main")
+-----------------------------------
+
+xi = xi or {}
+xi.avatarsFavor = xi.avatarsFavor or {}
+
+xi.avatarsFavor.skillLevels =
+{
+    [1] = 316,
+    [2] = 381,
+    [3] = 446,
+    [4] = 511,
+    [5] = 574,
+    [6] = 669,
+    [7] = 670,
+}
+
+-- Based on BGWiki Power 8 -> 10 is only available with +Avatars Favor Gear
+-- Avatar Favor buffs scale per tick (~9 seconds) to a max value based on current summoning skill
+local avatarsFavorEffect =
+{
+    [xi.pet.id.CARBUNCLE] = -- Regen
+    {
+        scaling = {12, 14, 16, 18, 20, 21, 24, 26, 27, 28},
+        effect = xi.effect.CARBUNCLES_FAVOR
+    },
+    [xi.pet.id.FENRIR] = -- Magic Eva
+    {
+        scaling = {1, 2, 3, 4, 5, 7, 9, 12, 15, 18},
+        effect = xi.effect.FENRIRS_FAVOR
+    },
+    [xi.pet.id.IFRIT] = -- Double Attack
+    {
+        scaling = {12, 12, 15, 15, 23, 23, 24, 24, 25, 25},
+        effect = xi.effect.IFRITS_FAVOR
+    },
+    [xi.pet.id.TITAN] = -- Defense
+    {
+        scaling = {57, 62, 67, 72, 77, 82, 87, 92, 97, 102},
+        effect = xi.effect.TITANS_FAVOR
+    },
+    [xi.pet.id.LEVIATHAN] = -- Magic Accuracy
+    {
+        scaling = {1, 2, 3, 4, 5, 7, 9, 12, 15, 18},
+        effect = xi.effect.LEVIATHANS_FAVOR
+    },
+    [xi.pet.id.GARUDA] = -- Evasion
+    {
+        scaling = {12, 15, 18, 22, 25, 28, 31, 34, 37, 40},
+        effect = xi.effect.GARUDAS_FAVOR
+    },
+    [xi.pet.id.SHIVA] = -- Magic Attack
+    {
+        scaling = {15, 18, 21, 24, 27, 30, 33, 36, 39, 42},
+        effect = xi.effect.SHIVAS_FAVOR
+    },
+    [xi.pet.id.RAMUH] = -- Potency (Critical Hit+%)
+    {
+        scaling = {12, 14, 16, 18, 19, 21, 21, 23, 23, 24},
+        effect = xi.effect.RAMUHS_FAVOR
+    },
+    [xi.pet.id.DIABOLOS] = -- Refresh
+    {
+        scaling = {3, 4, 4, 5, 5, 5, 6, 7, 7, 8},
+        effect = xi.effect.DIABOLOSS_FAVOR
+    },
+    [xi.pet.id.CAIT_SITH] = -- Magic Defense
+    {
+        scaling = {1, 2, 3, 4, 5, 6, 8, 10, 12, 15},
```

Comment:
This scaling doesn't match what is known on bg wiki for the last 5 tiers. where did this come from? the known pattern seems to be similar to carbuncles so i'd follow that on a guess, perhaps..

`10, 12, 14, 16, 18, 19, 22, 24, 26, 27, 28`

I don't like guessing, but we at least have something to start with. It will likely be approximate.

Reply:
This was just: We don't have a cait sith to use for testing so I didn't update the #s. I've added correct numbers and siren plumbing for future implementations.